openml-datarobot/README.md

```diff
@@ -15,9 +15,9 @@ To import DataRobot models you need to place them in a special folder structure
 └── model.json
 ```
 
-\<model name\> can be any name you want to give to the folder containing the model. You will need to provide the path to this folder.
+\<model name\> can be any name you want to give to the folder containing the model. At the import, you will need to provide the path to this folder.
```

Comment:
this seems weird to me. why not something more direct like "the name of the root folder containing the model is free of choice" for instance?

Reply:
The \<model name\> is a "visual" hint, it makes it easier to identify in the tree what we are talking about. I think the text is clear this way.